php/php.editor/src/org/netbeans/modules/php/editor/verification/UnreachableStatementHint.java

```diff
@@ -246,12 +247,23 @@ public void visit(ContinueStatement node) {
         }
 
         @Override
-        public void visit(ThrowStatement node) {
+        public void visit(ExpressionStatement node) {
             if (CancelSupport.getDefault().isCancelled()) {
                 return;
             }
             super.visit(node);
-            processLastStatement(node);
+            // PHP 8.0: throw statement -> throw expression
```

Comment:
Usually, it is a good idea to add also a number of PR and/or issue.

Reply:
Will do it. Thank you!

Reply:
Great, thanks!